### DISCUSSION TITLE
Reiteration of the branch name as there is growing consensus on main

### DISCUSSION THREAD

**Discussion by RonnyPfannschmidt** [pytest-dev/pytest#8245] [General] [answered]
Hi everyone,
A while back it was not yet clear how the default branch name is going

Now that more and more projects go to main and the consent on github.com also seems to be main

I want to put the rename of master to main back onto our todo list.

**Comment by nicoddemus** [comment, marked as answer]
👍 from me.

**Comment by symonk** [comment]
Any outstanding tasks here @RonnyPfannschmidt I can help with? I notice the renaming has occurred as well as an open PR for `scripts/docs`.  Unsure if there is any other things to be done, but happy to help if so.

**Reply by RonnyPfannschmidt** [reply]
@symonk after my seconf main pr i beleive most things are solved, a keen eye on missed details or other repos would help
